.pre-commit-hooks.yaml

```diff
@@ -6,6 +6,9 @@
 -   id: ansible-lint
     name: Ansible-lint
     description: This hook runs ansible-lint.
-    entry: ansible-lint
+    entry: ansible-lint --force-color .
```

Comment:
`--force-color` shouldn't be necessary as of `pre-commit 1.19`

Reply:
in fact, it's the _wrong thing to do_ if you're using a CI system, etc. which doesn't have color escapes